app/views/events/_form.html.erb

```diff
@@ -38,7 +38,7 @@
 
   <div class="field">
     <%= form.label :description %>
-    <%= form.text_field :description, placeholder: "Provide a description for the event", :class => 'textField' %>
+    <%= form.text_area :description, placeholder: "Provide a description for the event", :class => 'textField' %>
```

Comment:
I am assuming this change is just a usability fix and not vital to the calendar?